Configure

```diff
@@ -812,6 +823,13 @@ if ($libs =~ /(^|\s)-Wl,-rpath,/
 	"***** any of asan, msan or ubsan\n";
 }
 
+if ($#seed_sources < 0) {
+    die "No seed config and no defaults set";
```

Comment:
Well, that's why this is a WIP. :)  But while I'm here, what's the best way to tell if I'm doing windows linux or vms as the config?  Look at $target?